webrtc.html

```diff
@@ -11770,14 +11772,23 @@ <h3>Perfect Negotiation Example</h3>
 signaling.onmessage = async ({data: {description, candidate}}) => {
   try {
     if (description) {
-      const offerCollision = description.type == "offer" &&
-                             (makingOffer || pc.signalingState != "stable");
+      // An offer may come in while we are busy processing SRD(answer).
+      // In this case, we will be in "stable" by the time the offer is processed
+      // so it is safe to chain it on our Operations Chain now.
+      const readyForOffer =
+          !makingOffer &&
+          (pc.signalingState == "stable" || isSettingRemoteAnswerPending);
+      const offerCollision = description.type == "offer" && !readyForOffer;
 
       ignoreOffer = !polite && offerCollision;
       if (ignoreOffer) {
         return;
       }
+      if (description.type == "answer") {
+        isSettingRemoteAnswerPending = true;
+      }
```

Comment:
I wonder if this is better?
```suggestion
      isSettingRemoteAnswerPending = description.type == "answer";
```